odoo/addons/base/models/res_partner.py

```diff
@@ -348,6 +359,11 @@ def _compute_commercial_company_name(self):
             p = partner.commercial_partner_id
             partner.commercial_company_name = p.is_company and p.name or partner.company_name
 
+    def _compute_company_registry(self):
```

Comment:
this compute is probably never called since the compute doesn't have any dependency